stable/express-gateway/Chart.yaml

```diff
@@ -9,5 +9,5 @@ maintainers:
 name: express-gateway
 sources:
 - https://github.com/expressgateway/express-gateway
-version: 0.2.0
-appVersion: 1.11.0
+version: 1.12.0
```

Comment:
following semver the major bump indicates that this is a non-compatible change.
Myabe 0.3.0?

Reply:
Well I wanted to go with 1.x since we're now up at full speed. Would it be a problem if the helm would follow the same versioning as Express Gateway?

Reply:
Sounds reasonable.

P.S: We're going to release EG 1.12.1 soon so I think we can close this PR and wait today to release the new one. Thanks!